package-lock.json

```diff
@@ -1,8 +1,1849 @@
 {
   "name": "link-check",
   "version": "4.5.3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Unrelated change from installing with NPM 7

Reply:
I can revert but it seems compatible isn't it?